ci/setup-build-test-gdk.ps1

```diff
@@ -27,50 +28,44 @@ Start-Event "get-unreal-engine" "command"
 &$PSScriptRoot"\get-engine.ps1" -unreal_path "$unreal_path"
 Finish-Event "get-unreal-engine" "command"
 
-Start-Event "symlink-gdk" "command"
-$gdk_in_engine = "$unreal_path\Engine\Plugins\UnrealGDK"
-New-Item -ItemType Junction -Name "UnrealGDK" -Path "$unreal_path\Engine\Plugins" -Target "$gdk_home"
-Finish-Event "symlink-gdk" "command"
```

Comment:
We would want for our plugin to run as an engine plugin, but this does not currently work (see https://improbableio.atlassian.net/browse/GV-121). We therefore symlink the GDK into the project, but _also_ into the Engine (from what I understand, hoping that it will work someday for some reason). Previously this double symlinking did not cause issues, but during work on this PR the "wrong" engine location was wrongly used for some configurations.